idaes_ui/fv/example.py

```diff
@@ -22,14 +23,36 @@
 _log.setLevel(logging.INFO)
 
 
-def fv_example():
+class StopFile:
+    def __init__(self):
+        filename = f"stop_fv_example.txt"
+        self._path = Path(filename)
+        with self._path.open("w") as f:
+            f.write("RUN\n")
+            f.write("Change word above to 'stop' to stop the example\n")
+
+    def stopped(self) -> bool:
+        line = ""
+        try:
+            with self._path.open("r") as f:
+                line = f.readline()
+        except IOError:
+            pass
+        return line.lower().strip() == "stop"
+
+
+def fv_example(headless):
+    sf = StopFile()
```

Comment:
I think we should consider adding something so that the `StopFile`  cleans up after itself. Maybe a context manager?

```py
class StopFile:
     def __init__(self):
          self._path = Path(...)
    def __enter__(self):
         self._path.write_text("RUN\nChange word above to 'stop' to stop the example\n")
    def __exit__(self, *exc_info):
         self._path.unlink()

with StopFile() as sf:
    try:
         while True:
             if sf.stopped(): break
```

Reply:
I actually like having it there